src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -557,6 +557,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('exclude_path', exclude_path_type)
         c.argument('include_path', include_path_type)
         c.argument('recursive', recursive_type)
+        c.argument('content_type', arg_group='Additional Flags', help="Specify content type of the file. "
+                   "If unspecified, imply no-guess-mime-type.")
```

Comment:
According to https://docs.microsoft.com/en-us/azure/storage/common/storage-ref-azcopy-copy#options

```suggestion
                   "If unspecified, imply --no-guess-mime-type.")
```